src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-detail.component.html

```diff
@@ -13,7 +13,7 @@ <h2><span jhiTranslate="artemisApp.fileUploadExercise.detail.title">File Upload
                     [exercise]="fileUploadExercise"
                 ></jhi-non-programming-exercise-detail-common-actions>
             </div>
-            <div *ngIf="fileUploadExercise.releaseDate != undefined && moment(fileUploadExercise.releaseDate).isBefore(moment())" class="mt-3">
```

Comment:
We should rather move the evaluation to the component and not expose the date API to the template

Reply:
this line is deleted, so I do not understand what you are referring to. In general, I created a discussion point in a separate confluence page for the issue with exposing the date API to the template.

Reply:
I probably misclicked there, I meant the replacement line one below. But yes, it's the same principle so it's solved with the confluence page for now.